### DISCUSSION TITLE
Display Web Page Analytics to the User

### DISCUSSION THREAD

**Discussion by ayodyln** [vercel/community#2488] [Help]
### Summary

I'm figuring out how to display page views on one of my web pages. My first idea was to use the `Web Analytics` provided by Vercel. But, I'm lost on displaying specific page data and can't find much info on this. Should I use custom events? Or is there a better way for this?

For this, I'm using SvelteKit.

**Comment by amyegan** [comment]
The [Analytics](https://vercel.com/docs/concepts/analytics#) tab currently provides a variety of ways to view visitor metrics, including a list of pages with their view counts. For deeper detail, Pro and Enterprise teams can [track custom events](https://vercel.com/docs/concepts/analytics/custom-events#) to count interactions, such as button clicks or form submissions. But custom event tracking is not necessary to see the view counts broken down by page.
